### PR TITLE
Feature/display brewery detail

### LINKED ISSUE #5: User Story | Details Page
- [ ] As a user, I can click a brewery card/title, and see that brewery’s details

### LINKED ISSUE #6: User Story | Router Integration
- [ ] As a user, I should be able to share a link to the brewery I am currently looking

### PR DESCRIPTION
**Pull request by stephanie-roe**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 

Implemented Router to show a brewery by its ID. 
Displayed brewery details on the DOM. 
Stylized the brewery details page. 

Were there any issues that arose? 
n/a

Is there anything you need from your teammates? 
n/a

Any other comments, questions, or concerns?
n/a

## Issues
Closes #5
Closes #6

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -1,64 +1,88 @@
...
 type State = {
-  breweries: Brewery[]
+  breweries: Brewery[],
+  // specificBrewery: {}
 }
 
 // Brewery is an object that has all of these keys, which have specified data types as their values
   // each element in our breweries state is one of these objects
-type Brewery = {
+export type Brewery = {
```
Comment (userigorgithub) [inline src/Components/App.tsx:20]: Great job teaching what ? does Nick!

File: src/Components/BreweryDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import "../Styles/BreweryDetails.css"
+import { Brewery } from "./App";
+
```
Comment (userigorgithub) [inline src/Components/BreweryDetails.tsx:4]: Adding Pick... interesting.

File: src/Components/BreweryDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+//     longitude: string,
+//     latitude: string,
+//     phone: string,
+//     website_url: string,
+//     updated_at: string,
+//     created_at: string
+//   }
+
+class BreweryDetails extends React.Component<DetailsProps, DetailsState> {
+    state: DetailsState ={
```
Comment (userigorgithub) [inline src/Components/BreweryDetails.tsx:32]: GJ filtering properties.

File: src/Components/BreweryDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        }
+      })
+      .then(data => {
+        this.setState({ brewery: data })
+      })
+      .catch(error => console.log("Error"))
+    }
+
+    render() {
+        return(
```
Comment (userigorgithub) [inline src/Components/BreweryDetails.tsx:62]: Great! Returning data that we want!

**Review by userigorgithub — APPROVED** [review]
Great job resolving issues and figuring out how TypeScript works with using different containers!

File: src/Components/BreweryDetails.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+            <div className="brewery-details">
+                <div className="details-card">
+                  <h2>{this.state.brewery.name}</h2>
+                  <button>{this.state.brewery.brewery_type}</button>
+                  <div className="contact-info">
+                    <p>{this.state.brewery.website_url}</p>
+                    <p>{this.state.brewery.phone}</p>
+                    <p>{this.state.brewery.street}</p>
+                    <p>{`${this.state.brewery.city}, ${this.state.brewery.state} ${this.state.brewery.postal_code}`}</p>
```
Comment (rachellhill) [inline src/Components/BreweryDetails.tsx:71]: This looks so good!

File: src/Components/App.tsx [hunk 6]
```diff
@@ -1,64 +1,88 @@
...
     return (
-      <div className='app'>
+      <main className='app'>
         <h1>Brew Maps</h1>
-        <Breweries newBrewery={breweries}/>
-      </div>
+        <Route exact path="/">
+            <Breweries newBrewery={breweries} />
+        </Route>
+        <Route path="/:id" render={ ({match}) => <BreweryDetails id={match.params.id} /> } >
```
Comment (rachellhill) [inline src/Components/App.tsx:83]: Nice job using this!

**Review by rachellhill — APPROVED** [review]
great work team!

**Pull request merged**